nss-day-cohort-60/national-parks-client
# Hp park page

# Description
Created a new page to view individualized information for each park

# Steps to test
1. `git fetch origin hp-park-page`
2. `git checkout hp-park-page`
3. Start/restart debugger
4. Go to http://localhost:3000/home and click on the name of a park
5. Or just go to http://localhost:3000/parks/3
6. Do you see a photo carousel? How about some wildlife? Some camping info? A list of amenities? If you answered yes to all of these questions, this branch is fully functional.
